litellm/model_prices_and_context_window_backup.json

```diff
@@ -1800,6 +1800,26 @@
         "supports_vision": true,
         "source": "https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#foundation_models"
     },
+    "medlm-medium": {
+        "max_tokens": 8192,
+        "max_input_tokens": 32768,
+        "max_output_tokens": 8192,
+        "input_cost_per_character": 0.0000005,
```

Comment:
hey @skucherlapati where is the pricing for med-lm? is it just hidden for me b/c of access issues ? 
<img width="981" alt="Screenshot 2024-07-17 at 4 49 17 PM" src="https://github.com/user-attachments/assets/5bea5bf5-2607-43d9-a733-a80b43854d5c">

Reply:
Its on a different link. I added it as Reference in PR description. Also adding it below for easy access:
https://cloud.google.com/vertex-ai/generative-ai/docs/medlm/pricing

Reply:
Are there any optional params I can add it to in the json for future reference if pricing changes?

Reply:
yes - it's the `source` field

Reply:
So there are two sources here: token usage source and pricing source pages. The one added in the PR is for token usage and I think that makes most sense because any major version updates would appear there. Unless there is an additional key `other than "source"` we will have to do with what we have the json for now. let me know if this works?

Reply:
looks good